mlflow/entities/model_registry/registered_model_detailed.py

```diff
@@ -1,10 +1,15 @@
 from mlflow.entities.model_registry.model_version_detailed import ModelVersionDetailed
 from mlflow.entities.model_registry.registered_model import RegisteredModel
 from mlflow.protos.model_registry_pb2 import RegisteredModelDetailed as ProtoRegisteredModelDetailed
+from mlflow.utils import experimental
 
 
+@experimental
 class RegisteredModelDetailed(RegisteredModel):
-    # __init__ method to initialize fields
+    """
+    MLflow entity for Registered Model Detailed
```

Comment:
do these actually appear in the docs? if so it may be worth saying how it is different from registered model

Reply:
same for model version detailed

Reply:
I think we would have to include these, since these are returned by APIs.

Reply:
makes sense. i wasn’t sure if they were correctly exposed for docs.

Reply:
I will add details in doc strings to clarify between non-`Detailed` and `*Detailed` entities.